bgc_data/bgc_pigments.dataresource.yaml

```diff
@@ -18,14 +18,14 @@ schema:
   - name: SAMPLEDATELOCAL
     title: "Date & time of sample in local time (formatted string)"
     type: datetime
-    # format string *must* be specified using standard Python/C strptime specification         
-    # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior               
+    # format string *must* be specified using standard Python/C strptime specification
+    # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior
     format: "%Y-%m-%d %H:%M:%S"
-    required: false
+    required: true
   - name: SAMPLEDEPTH_M
-    title: "Depth (m) that sample was obtained"
+    title: "Depth (m) that sample was obtained, or 'WC' if combined water-column sample"
     type: string
-    required: false
+    required: true
```

Comment:
If SAMPLEDEPTH_M is part of the primary key, it must be required

Reply:
agree